composites/Plugin/ContentAnalysis/tests/ContentAnalysisTest.js

```diff
@@ -53,199 +53,223 @@ const considerationsResults = [
+				changeLanguageLink={ "#" }
+				language="English"
+			/>
+		);
 
-test( "the ContentAnalysis component with disabled buttons matches the snapshot", () => {
-	const component = createComponentWithIntl(
-		<ContentAnalysis
-			problemsResults={ problemsResults }
-			improvementsResults={ improvementsResults }
-			goodResults={ goodResults }
-			considerationsResults={ considerationsResults }
-			errorsResults={ errorsResults }
-			changeLanguageLink={ "#" }
-			language="English"
-			showLanguageNotice={ true }
-			marksButtonStatus={ "disabled" }
-		/>
-	);
-
-	let tree = component.toJSON();
-	expect( tree ).toMatchSnapshot();
-} );
+		let tree = component.toJSON();
+		expect( tree ).toMatchSnapshot();
+	} );
+
+	it( "the ContentAnalysis component with specified header level matches the snapshot", () => {
+		const component = createComponentWithIntl(
+			<ContentAnalysis
+				headingLevel={ 3 }
+				problemsResults={ problemsResults }
+				improvementsResults={ improvementsResults }
+				goodResults={ goodResults }
+				considerationsResults={ considerationsResults }
+				errorsResults={ errorsResults }
+				changeLanguageLink={ "#" }
+				language="English"
+			/>
+		);
+
+		let tree = component.toJSON();
+		expect( tree ).toMatchSnapshot();
+	} );
+
+	it( "the ContentAnalysis component with language notice matches the snapshot", () => {
+		const component = createComponentWithIntl(
+			<ContentAnalysis
+				problemsResults={ problemsResults }
+				improvementsResults={ improvementsResults }
+				goodResults={ goodResults }
+				considerationsResults={ considerationsResults }
+				errorsResults={ errorsResults }
+				changeLanguageLink={ "#" }
+				language="English"
+				showLanguageNotice={ true }
+			/>
+		);
+
+		let tree = component.toJSON();
+		expect( tree ).toMatchSnapshot();
+	} );
+
+	it( "the ContentAnalysis component with language notice for someone who can change the language matches the snapshot", () => {
+		const component = createComponentWithIntl(
+			<ContentAnalysis
+				problemsResults={ problemsResults }
+				improvementsResults={ improvementsResults }
+				goodResults={ goodResults }
+				considerationsResults={ considerationsResults }
+				errorsResults={ errorsResults }
+				changeLanguageLink={ "#" }
+				language="English"
+				showLanguageNotice={ true }
+				canChangeLanguage={ true }
+			/>
+		);
+
+		let tree = component.toJSON();
+		expect( tree ).toMatchSnapshot();
+	} );
+
+	it( "the ContentAnalysis component with language notice for someone who cannot change the language matches the snapshot", () => {
+		const component = createComponentWithIntl(
+			<ContentAnalysis
+				problemsResults={ problemsResults }
+				improvementsResults={ improvementsResults }
+				goodResults={ goodResults }
+				considerationsResults={ considerationsResults }
+				errorsResults={ errorsResults }
+				changeLanguageLink={ "#" }
+				language="English"
+				showLanguageNotice={ true }
+				canChangeLanguage={ true }
```

Comment:
Shouldn't this be false? it's testing for "someone who cannot change the language"

Reply:
Nice spot. That makes complete sense to me yes. Changing.